src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsPartnerInfo.tsx

```diff
@@ -0,0 +1,108 @@
+import { FollowProfileButtonQueryRenderer } from "Components/FollowButton/FollowProfileButton"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkDetailsPartnerInfoQuery } from "__generated__/ArtworkDetailsPartnerInfoQuery.graphql"
+import { ArtworkDetailsPartnerInfo_artwork$data } from "__generated__/ArtworkDetailsPartnerInfo_artwork.graphql"
+import { ContextModule } from "@artsy/cohesion"
+import { Skeleton, SkeletonBox, StackableBorderBox } from "@artsy/palette"
+import { EntityHeaderPartnerFragmentContainer } from "Components/EntityHeaders/EntityHeaderPartner"
+import { useSystemContext } from "System"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+
+export interface PartnerInfoProps {
+  artwork: ArtworkDetailsPartnerInfo_artwork$data
+}
+
+export const PartnerInfo: React.FC<PartnerInfoProps> = ({ artwork }) => {
+  const { partner } = artwork
+
+  if (!partner) return null
+
+  const canLink = Boolean(
+    partner?.partnerPageEligible && partner?.isDefaultProfilePublic
+  )
+
+  return (
+    <StackableBorderBox flexDirection="column" data-test="aboutTheWorkPartner">
```

Comment:
**question**: should we use `StackableBorderBox` here?

Reply:
yes we should because if we don't we get this result:

<img width="1061" alt="Screenshot 2022-12-19 at 12 05 49" src="https://user-images.githubusercontent.com/21178754/208412350-2d82c492-5c24-4fc2-8d84-efca2d7bdd68.png">